src/main/java/TaskList.java

```diff
@@ -0,0 +1,135 @@
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+
+public class TaskList {
+    List<Task> taskList;
+    Storage storage;
+
+    public TaskList(Storage storage) {
+        this.storage = storage;
+        this.taskList = storage.loadTask();
+    }
+
+    /**
+     * Adds task of taskName to taskList.
+     * @param taskName Task to be added to taskList
+     */
+    public void addTask(String taskName) {
+        try {
+            String taskType = taskName.split(" ", 2)[0];
+            String taskDesc = taskName.split(" ", 2)[1];
```

Comment:
I think it's possible to handle the string splitting in the Parser class instead. You can consider passing a Task as a parameter in place of the String taskName! :)